src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -1491,7 +1491,7 @@ def iloc():
         With a scalar integer.
 
         >>> type(df.iloc[0])
-        <class 'snowflake.snowpark.modin.pandas.series.Series'>
+        <class 'modin.pandas.series.Series'>
```

Comment:
i felt little bit wired to have test shown the whole class path in the example, is that the only place we have this? we can fix it like this for now, but maybe later we can have a separate pr to remove those in all examples